scss/_utilities_grid.scss

```diff
@@ -0,0 +1,19 @@
+@mixin grid-first-col {
+  .u-first-col--mobile {
```

Comment:
What is the use case for these?

Reply:
They are for selecting the first column of wrapping items in lists, from what I understand.

i.e., if you have 8 items on a large viewport and two rows of 4 items wrapping in a small viewport then the 5th item needs the `.u-first-col--mobile` (I agree with your other comment, it should be `.u-first-col--small`) to fix the first-item specific styling.